gradle.properties

```diff
@@ -1,5 +1,5 @@
-version=gramine
-iexecCommonVersion=gramine-NEXT-SNAPSHOT
+version=7.1.0
+iexecCommonVersion=6.0.1
```

Comment:
`-NEXT-SNAPSHOT` version for `iexec-common` ?

Reply:
```suggestion
iexecCommonVersion=6.0.1-NEXT-SNAPSHOT
```